ambari/manifest.json

```diff
@@ -11,8 +11,7 @@
   "support": "core",
   "supported_os": [
     "linux",
-    "mac_os",
-    "windows"
```

Comment:
why is windows not supported if its just making http calls?

Reply:
I dunno 🤷‍♂ their docs say no Windows support so may give customers the wrong idea. I can add back if you want

Reply:
Yeah, this is a weird integration, it does not need to be installed on the same machine as the thing that is integrating with